### PR TITLE
pan/types: Cleanup style - lint, indentation, wrapping

### PR DESCRIPTION
**Pull request by jrha**
Addresses some of the issues flagged by tests (but that were not part of) #196.

### CODE REVIEW

File: pan/types.pan [hunk 1]
```diff
@@ -48,12 +48,13 @@ Ex: 20040825120123Z
 }
 function is_asndate = {
     # Check cardinality and type of argument.
-    if (ARGC != 1 || !is_string(ARGV[0]))
-        error("usage: is_asndate(string)");
+    if (ARGC != 1 || !is_string(ARGV[0])) error("usage: is_asndate(string)");
```
Comment (ned21) [inline pan/types.pan]: Really not a fan of this style. 

```
if (...) 
    # action;
```

is much more readable most of the time IMHO, especially when the $condition is contains multiple operators. As a code reader I want to be able to focus on the logic and be able to visually filter out the resulting action.

Comment (jrha) [reply pan/types.pan]: Would the following be okay?
```pan
if (...) {
    # action;
};
```

Comment (ned21) [reply pan/types.pan]: Even better.

**Comment by jrha** [comment]
I'll rebase #196.

**Pull request merged**